### PR TITLE
DockerCompose.yaml changes

### PR DESCRIPTION
**Pull request by isaac73**
# Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] All new and existing tests passed.
- [x] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change
Changes to yaml file after cache restructure
...

# Motivation for the change

...

### CODE REVIEW

File: scripts/docker/docker-compose.yml [hunk 1]
```diff
@@ -95,6 +95,8 @@ services:
     image: azureiotpcs/pcs-config-java:latest
     depends_on:
       - storageadapter
+      - devicesimulation
```
Comment (timlaverty) [inline scripts/docker/docker-compose.yml]: isn't this a circular dependency?

Comment (isaac73) [reply scripts/docker/docker-compose.yml]: No. deviceSimulation doesn't depend on config service.

**Comment by dluc** [comment]
@isaac73 pls reply to comments for the record, and see if we can unblock/merge

**Comment by isaac73** [comment]
@timlaverty config depends on Device Simulation for seeding simulated devices and Telemetry for seeding rules. No, there isn't any circular dependency here.

**Pull request merged**